docs/content/s3.md

```diff
@@ -2249,6 +2250,72 @@ So once set up, for example to copy files into a bucket
 rclone copy /path/to/files minio:bucket
 ```
 
+### RackCorp {#RackCorp}
+
+[RackCorp Object Storage](https://www.rackcorp.com/storage/s3storage) is an S3 compatible object storage platform from your friendly cloud provider RackCorp. The service is fast, reliable, well priced and located in many strategic locations unserviced by others, to ensure you can maintain data sovereignty.
+
+Before you can use RackCorp Object Storage, you'll need to "[sign up](https://www.rackcorp.com/signup)" for an account on our "[portal](https://portal.rackcorp.com)", where you can create an `access key`, a `secret key` and `buckets`, in your location of choice with ease. These details are required for the next steps of configuration, when `rclone config` asks for your `access_key_id` and `secret_access_key`.
+
+At present, the following `region` and `location_constraint` combinations are available. They are only relevant at the time of creating a bucket and will indicate the location or region where your data will be stored.
```

Comment:
Can you remove these lists of region and location_constraint? I see them getting out of date with respect to the actual configurator?

Reply:
Done @ncw; thanks for being so patient with me :)